kf_model_fhir/mappers/resources/kfdrc_vital_status.py

```diff
@@ -0,0 +1,79 @@
+"""
+This module converts Kids First Outcomes to FHIR kfdrc-vital-status-no-phi
+(derived from FHIR Observation).
+"""
+from kf_lib_data_ingest.common import constants
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+from common.utils import make_identifier, make_select, get
+
+RESOURCE_TYPE = 'Observation'
+
+interpretation = {
+    constants.OUTCOME.VITAL_STATUS.ALIVE: True,
+    constants.OUTCOME.VITAL_STATUS.DEAD: False,
+    constants.COMMON.UNKNOWN: {
+        'system': 'http://terminology.hl7.org/CodeSystem/v3-NullFlavor',
+        'code': 'NI',
+        'display': 'No information'
+    }
+}
+
+def yield_kfdrc_vital_statuses(eng, table, study_id, kfdrc_patients):
+    for row in make_select(
+            eng, table,
+            CONCEPT.PARTICIPANT.ID,
+            CONCEPT.OUTCOME.EVENT_AGE_DAYS,
+            CONCEPT.OUTCOME.VITAL_STATUS
+        ):
+        participant_id = get(row, CONCEPT.PARTICIPANT.ID)
+        event_age_days = get(row, CONCEPT.OUTCOME.EVENT_AGE_DAYS)
+        vital_status = get(row, CONCEPT.OUTCOME.VITAL_STATUS)
+        status = interpretation.get(vital_status)
+
+        if (not participant_id or not status):
+            continue
+
+        retval = {
+            'resourceType': RESOURCE_TYPE,
+            'id': make_identifier(RESOURCE_TYPE, study_id, participant_id, name, observed, event_age_days),
+            'meta': {
+                'profile': [
+                    'http://fhir.kids-first.io/StructureDefinition/kfdrc-vital-status-no-phi'
```

Comment:
Hey @gsantia not sure if you're ready for a review but figured I'd comment on this now since we know it has to change. Because of the server issue mentioned here: https://github.com/kids-first/kf-api-fhir-service/issues/35, 
can you change this to the `kfdrc-vital-status` profile? Also make sure to update the module docstring ^

Reply:
@znatty22, I will make the same changes to other mappers.
@gsantia, per the ID creation logic, please refer to https://github.com/kids-first/kf-lib-data-ingest/blob/master/kf_lib_data_ingest/target_apis/kids_first.py#L478-L484

Reply:
> Hey @gsantia not sure if you're ready for a review but figured I'd comment on this now since we know it has to change. Because of the server issue mentioned here: [kids-first/kf-api-fhir-service#35](https://github.com/kids-first/kf-api-fhir-service/issues/35),
> can you change this to the `kfdrc-vital-status` profile? Also make sure to update the module docstring ^

Done, thanks!



> @znatty22, I will make the same changes to other mappers.
> @gsantia, per the ID creation logic, please refer to https://github.com/kids-first/kf-lib-data-ingest/blob/master/kf_lib_data_ingest/target_apis/kids_first.py#L478-L484

thanks I have no idea why I uploaded such an incorrect ID creation call! fixed